packages/liveblocks-react/package.json

```diff
@@ -35,10 +35,10 @@
   "dependencies": {
     "@liveblocks/client": "1.3.0",
     "@liveblocks/core": "1.3.0",
-    "nanoid": "^4.0.2",
     "use-sync-external-store": "^1.2.0"
   },
   "peerDependencies": {
+    "nanoid": "^3 || ^4",
```

Comment:
Oh, that's clever. That lets people decide which one to use! I like it 👍

Reply:
The issue I have with that is that if `^4` is picked (e.g. automatically), our CJS bundle might be incorrect depending on the final bundler or lack of it. I think the ideal setup is to add it as a normal dependency as `"nanoid": "^3.3.6"`.